components/server/src/data/datamodel.json

```diff
@@ -4732,13 +4732,13 @@
         "generic": {
             "name": "Generic Security Warning",
             "description": "Generic Security Warning vulnerability report in JSON format",
-            "url": "",
+            "url": "https://www.google.com",
```

Comment:
This should be a link to a specification of the format, maybe a separate markdown file in the docs folder?